doc/i2p.md

```diff
@@ -166,3 +166,13 @@ In most cases, the default router settings should work fine.
 
 Please see the "General Guidance for Developers" section in https://geti2p.net/en/docs/api/samv3
 if you are developing a downstream application that may be bundling I2P with Bitcoin.
+
+## Privacy recommendations
+
+- Operating a node that listens on multiple networks (e.g. IPv4 and I2P) can help
```

Comment:
While the study linked to in the OP described IPv4 and Tor, I suspect the most frequent multiple network pairing would be Tor and I2P, as privacy network users might opt to avoid clearnet (say, the kind of node operator that pre-I2P may have been running over Tor only).
  
    
      
        Suggested change
      
    
    
      
          
            
            - Operating a node that listens on multiple networks (e.g. IPv4 and I2P) can help
          
          
            
            - Operating a node that listens on multiple networks (e.g. I2P and Tor) can help

Reply:
> This combination, IMO, poses less risk to the privacy-concerned individual as they are exposing their node over two privacy networks.

Perhaps clarify that in the documentation, as those that do run over both Tor and I2P could conclude that they risk being fingerprinted in the same way as described here.

I could be wrong, but I don't see who would listen on both clearnet and Tor (or I2P) if they are concerned about privacy. ISTM those who do that are willingly providing a service to the network or gathering data/doing research.

Perhaps also provide explicit workarounds, rather than hinting at them, like turning off listening, or running two separate nodes each on one network that connect manually to each other.

Reply:
> I could be wrong, but I don't see who would listen on both clearnet and Tor (or I2P) if they are concerned about privacy.

Perhaps not everybody understands it as well as you do. The purpose of this PR is to more explicitly inform people who aren't fully aware of the risks.

Reply:
> Perhaps clarify that in the documentation, as those that do run over both Tor and I2P could conclude that they risk being fingerprinted in the same way as described here.

Would be good to address this as well.